statsmodels/tools/sequences.py

```diff
@@ -0,0 +1,213 @@
+
+
+def n_primes(n):
+    """List of the n-first prime numbers.
+
+    Parameters
+    ----------
+    n : int
+        Number of prime numbers wanted.
+
+    Returns
+    -------
+    primes : list(int)
+        List of primes.
+
+    """
+    primes = [2, 3, 5, 7, 11, 13, 17, 19, 23, 29, 31, 37, 41, 43, 47, 53, 59,
+              61, 67, 71, 73, 79, 83, 89, 97, 101, 103, 107, 109, 113, 127,
+              131, 137, 139, 149, 151, 157, 163, 167, 173, 179, 181, 191, 193,
+              197, 199, 211, 223, 227, 229, 233, 239, 241, 251, 257, 263, 269,
+              271, 277, 281, 283, 293, 307, 311, 313, 317, 331, 337, 347, 349,
+              353, 359, 367, 373, 379, 383, 389, 397, 401, 409, 419, 421, 431,
+              433, 439, 443, 449, 457, 461, 463, 467, 479, 487, 491, 499, 503,
+              509, 521, 523, 541, 547, 557, 563, 569, 571, 577, 587, 593, 599,
+              601, 607, 613, 617, 619, 631, 641, 643, 647, 653, 659, 661, 673,
+              677, 683, 691, 701, 709, 719, 727, 733, 739, 743, 751, 757, 761,
+              769, 773, 787, 797, 809, 811, 821, 823, 827, 829, 839, 853, 857,
+              859, 863, 877, 881, 883, 887, 907, 911, 919, 929, 937, 941, 947,
+              953, 967, 971, 977, 983, 991, 997][:n]
+
+    if len(primes) < n:
+        big_number = 10
+        while 'Not enought primes':
+            primes = primes_from_2_to(big_number)[:n]
+            if len(primes) == n:
+                break
+            big_number += 1000
+
+    return primes
+
+
+def van_der_corput(n_sample, base=2, start_index=0):
+    """Van der Corput sequence.
+
+    Pseudo-random number generator based on a b-adic expansion.
+
+    Parameters
+    ----------
+    n_sample : int
+        Number of element of the sequence.
+    base : int
+        Base of the sequence.
+    start_index : int
+        Index to start the sequence from.
+
+    Returns
+    -------
+    sequence : list (n_samples,)
+        Sequence of Van der Corput.
+
+    """
+    sequence = []
+    for i in range(start_index, start_index + n_sample):
+        n_th_number, denom = 0., 1.
+        quotient = i
+        while quotient > 0:
+            quotient, remainder = divmod(quotient, base)
+            denom *= base
+            n_th_number += remainder / denom
+        sequence.append(n_th_number)
+
+    return sequence
+
+
+def halton(dim, n_sample, feature_range=None, start_index=0):
+    """Halton sequence.
+
+    Pseudo-random number generator that generalize the Van der Corput sequence
+    for multiple dimensions. Halton sequence use base-two Van der Corput
+    sequence for the first dimension, base-three for its second and base-n for
+    its n-dimension.
+
+    Parameters
+    ----------
+    dim : int
+        Dimension of the parameter space.
+    n_sample : int
+        Number of samples to generate in the parametr space.
+    feature_range : tuple or array_like ([min, k_vars], [max, k_vars])
+        Desired range of transformed data.
```

Comment:
this should also mention that there will be points at the boundary. 
The transformation uses min and max of sample and not the theoretical space, unit cube.